ecosystem/sep-0006.md

```diff
@@ -1426,6 +1426,7 @@ Each object in the `transactions` array should have the following fields:
 | `started_at`              | UTC ISO 8601 string | (optional) Start date and time of transaction.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | `updated_at`              | UTC ISO 8601 string | (optional) The date and time of transaction reaching the current status.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
 | `completed_at`            | UTC ISO 8601 string | (optional) Completion date and time of transaction.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| `action_required_by`      | UTC ISO 8601 string | (optional) The date and time by when the user action is required. This field must be omitted if no user action is required in the current status. Anchor should automatically move transaction into the next status (such as `expired`) once the deadline has been reached.                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
I really like your description of `action_required_by` you wrote for the PR description, can we use that (or parts of it) here?

Reply:
Can this name be more specific like for example `user_action_required_by`?

Reply:
That was actually one of the names we were discussing, I don't have any strong preference, so let's rename it so it's a bit more clear!